packages/screens/LaunchpadERC20/LaunchpadERC20Sales/LaunchpadERC20CreateSale.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { View } from "react-native";
+
+import { CreateSaleForm } from "./LaunchpadERC20CreateSaleForm";
+import { CreateSaleSign } from "./LaunchpadERC20CreateSaleSign";
+import { LaunchpadERC20CreateSaleHeader } from "../component/LaunchpadERC20CreateSaleHeader";
+import { useCreateSaleState } from "../hooks/useCreateSale";
+
+import { Breadcrumb } from "@/components/Breadcrumb";
+import { ScreenContainer } from "@/components/ScreenContainer";
+import { SpacerColumn } from "@/components/spacer";
+import { NetworkKind } from "@/networks";
+import { ScreenFC } from "@/utils/navigation";
+
+const renderStep = (stepIndice: number) => {
+  if (stepIndice === 1) return <CreateSaleForm />;
+  if (stepIndice === 2) return <CreateSaleSign />;
+};
+
+const STEPS = ["Informations", "Sign & Deploy"];
+
+export const LaunchpadERC20CreateSaleScreen: ScreenFC<
+  "LaunchpadERC20CreateSale"
+> = () => {
+  const {
+    stepIndice,
+    actions: { gotoStep },
+  } = useCreateSaleState();
+
+  return (
+    <ScreenContainer
+      forceNetworkKind={NetworkKind.Gno}
+      isLarge
+      responsive
+      headerChildren={<LaunchpadERC20CreateSaleHeader />}
```

Comment:
Please, modify `headerChildren` and `onBackPress` like this for all `LaunchpadERC20 Sales, Tokens and Airdrop` subscreens.

You can delete your Header components imho.

```suggestion
      forceNetworkKind={NetworkKind.Gno}
      isLarge
      responsive
      headerChildren={
        <BrandText style={fontSemibold20}>Launchpad ERC20 Sale Creation</BrandText>
      }
      onBackPress={() => navigation.navigate("LaunchpadERC20Sales")}
```

Reply:
[c84dc0a](https://github.com/TERITORI/teritori-dapp/pull/1276/commits/c84dc0a9f371a9d62f76cccb41042633ac7ea311)